packages/date-time/etc/date-time.api.md

```diff
@@ -121,8 +123,8 @@ export interface ICalendarDayProps extends IBaseProps_2<ICalendarDay>, ICalendar
 // @public (undocumented)
 export interface ICalendarFormatDateCallbacks {
     formatDay: (date: Date) => string;
-    formatMonthDayYear: (date: Date, strings?: ICalendarStrings) => string;
-    formatMonthYear: (date: Date, strings?: ICalendarStrings) => string;
+    formatMonthDayYear: (date: Date, strings?: IGridStrings) => string;

```

Comment:
@lorejoh12 please I've limited the localization strings object we are sending to formatters to have only `days`, `months`, `shortDays` and `shortMonths` - would that be OK?
Or should I add some other fields there?

Reply:
it seems reasonable to me. If we need additional strings from the full object for later formatters we'll add them.